tests/utils/varUtils.ts

```diff
@@ -0,0 +1,25 @@
+import * as fs from "fs";
+import path from "path";
+
+const VARFILE_EXTENSION = ".zzv";
+
+function getVarFilePaths(dirPath: string): string[] {
```

Comment:
Let us not read extra variables from files. These probably will go into the testing of the runner rather than zzapi itself.

The idea is to simplify test code. If the test code itself is not simple, when something fails, we will not be sure if the test code has a bug or the module has a bug.

Try and simplify it as much as you can. Bare minimum, only to test the functionality that this module provides, nothing more.

Reply:
Loading variables from varFiles is still the job of the module right? The runner just gets the content and passes it to the module. 

I will attempt to simplify the code.